pandas/core/tools/numeric.py

```diff
@@ -88,7 +89,8 @@ def to_numeric(
         the dtype it is to be cast to, so if none of the dtypes
         checked satisfy that specification, no downcasting will be
         performed on the data.
-    dtype_backend : {'numpy_nullable', 'pyarrow'}, default 'numpy_nullable'
+
+    dtype_backend : {'numpy_nullable', 'pyarrow'}, default not nullable numpy
```

Comment:
I believe the logic should be as follows:

The default setting is NO_DEFAULT, which will behave like we do before, using `float64` for `nan` values. If the value is not `nan`, it will infer `int64` or similar. I’ve named this the `not nullable numpy` temporarily (perhaps it should be called `default-numpy`?).